lib/Widgets/TimePicker.vala

```diff
@@ -193,35 +238,65 @@ namespace Granite.Widgets {
          * @param format_24 The desired custom 24h format. For example "%H:%M".
          */
         public TimePicker.with_format (string format_12, string format_24) {
-            Object (format_12: format_12, format_24: format_24);
+            Object (custom_format_12: format_12, custom_format_24: format_24);
+        }
+
+        /**
+         * Creates a new TimePicker.
+         *
+         * @param with_seconds Should the format include seconds. For example: "%l:%M:%S %p" or "%H:%M:%S".
+         */
+        public TimePicker.with_second (bool with_second) {
```

Comment:
I don't think this should take an argument. It probably doesn't make sense to say `TimePicker.with_seconds (false)`. That would just be the same as doing `TimePicker ()`

Reply:
Absolutely. Done.